c++/src/kj/refcount.h

```diff
@@ -115,6 +142,102 @@ Own<T> Refcounted::addRefInternal(T* object) {
   return Own<T>(object, *refcounted);
 }
 
+template <typename T>
+Rc<T> Refcounted::addRcRefInternal(T* object) {
+  Refcounted* refcounted = object;
+  ++refcounted->refcount;
+  return Rc<T>(object);
+}
+
+template<typename T>
+class Rc: public Own<T> {
+  // Smart pointer for reference counted objects. 
+  //
+  // There are only two ways to obtain new Rc instances:
+  // - use kj::rc<T>(...) function to create new T.
```

Comment:
```suggestion
  // - use kj::Rc<T>(...) function to create new T.
```

Reply:
It is actually `kj::rc`.

Reply:
oh! my apologies. I misread this was a function.